plugins/woocommerce/tests/e2e-pw/tests/admin-marketing/overview.spec.js

```diff
@@ -27,11 +27,6 @@ test.describe( 'Marketing page', () => {
 		).toBeVisible();
 
 		// Sections present
-		await expect(
-			page.getByText(
-				'Reach new customers and increase sales without leaving WooCommerce'
-			)
-		).toBeVisible();
```

Comment:
Removing this part since it's covered in the next test, `Introduction can be dismissed`. 

Intro is already dismissed from the Pressable website and there's no easy way (API) to put it there before every execution.